src/containers/pages/IRS/Map/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -151,51 +151,34 @@ Array [
 exports[`components/IRS Reports/IRSReportingMap renders correctly: Response item ProgressBar 1`] = `
 Array [
   <ProgressBar
+    animate={false}
+    cssClass="progress-bar-striped"
     decimalPoints={0}
     height="10px"
-    indicatorThresholds={
+    lineColor="#0000FF"
+    lineColorThresholds={
       Object {
-        "GREEN_THRESHOLD": Object {
-          "color": "#2ECC40",
-          "name": "Green",
-          "value": 1,
-        },
-        "GREY_THRESHOLD": Object {
-          "color": "#dddddd",
-          "name": "Grey",
-          "value": 0.2,
-        },
-        "RED_THRESHOLD": Object {
-          "color": "#FF4136",
-          "name": "Red",
-          "orEquals": true,
-          "value": 0.75,
-        },
-        "YELLOW_THRESHOLD": Object {
-          "color": "#FFDC00",
-          "name": "Yellow",
-          "value": 0.9,
-        },
```

Comment:
Do you think there is value in maintaining these older style thresholds?  For example, how would we now deal with a threshold that is `>= 80%`?

Reply:
if it's greater than the color value then it takes the default lineColor which is "#0000FF"

Reply:
@RayceeM I mean how will we handle the possibility of having `>=` as well as just `>` for the thresholds?